test/WebSites/ApiExplorerWebSite/Controllers/ApiExplorerResponseTypeOverrideOnActionController.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace ApiExplorerWebSite
 {
-    [Produces("*/*", Type = typeof(Product))]
+    [Produces("applcation/json", Type = typeof(Product))]
```

Comment:
thanks for catching, nope it did not .. the reason is if it does not recognize this it will fall back to default which is json.